docs/book/developer-guide/stacks-profiles-repositories/stacks_profiles_repositories.md

```diff
@@ -0,0 +1,539 @@
+
+```shell
+zenml <STACK_COMPONENT> list
+```
+
+In order to see all the *available* flavors for a specific stack component, use:
+
+```shell
+zenml <STACK_COMPONENT> flavor list
+```
+
+{% hint style="info" %}
+Our CLI features a wide variety of commands that let you manage and use your stacks.
+If you would like to learn more, please run: "`zenml stack --help`"
+or visit [our CLI docs](https://apidocs.zenml.io/latest/cli/).
+{% endhint %}
+
+### Registering New Stacks
+
+You can combine various MLOps tools into a ZenML stack as follows:
+
+1. [Register a stack component](#registering-stack-components) for each tool 
+using `zenml <STACK_COMPONENT> register`,
+2. [Register a stack](#registering-a-stack) to bring all tools together using
+`zenml stack register`,
+3. [Activate the stack](#activating-a-stack) using `zenml stack set`. Now all
+your code is automatically executed using the desired tools / infrastructure.
+
+#### Registering Stack Components
+
+First, you need to create a new instance of the respective stack component
+with the desired flavor using `zenml <STACK_COMPONENT> register <NAME> --flavor=<FLAVOR>`. 
+Most flavors require further parameters that you can pass as additional
+arguments `--param=value`, similar to how we passed the flavor.
+
+E.g., to register a *local* artifact store, we could use the following command:
+
+```shell
+zenml artifact-store register <ARTIFACT_STORE_NAME> --flavor=local --path=/path/to/your/store
+```
+
+Afterwards, you should be able to see the new artifact store in the
+list of registered artifact stores, which you can access using the following command:
+
+```shell
+zenml artifact-store list
+```
+
+{% hint style="info" %}
+Our CLI features a wide variety of commands that let you manage and use your
+stack components and flavors. If you would like to learn more, please run
+`zenml <STACK_COMPONENT> --help` or visit [our CLI docs](https://apidocs.zenml.io/latest/cli/).
+{% endhint %}
+
+#### Registering a Stack
+
+After registering each tool as the respective stack component, you can combine
+all of them into one stack using the `zenml stack register` command:
+
+```shell
+zenml stack register <STACK_NAME> --orchestrator <ORCHESTRATOR_NAME> \
+                                  --artifact-store <ARTIFACT_STORE_NAME> \
+                                  --metadata-store <METADATA_STORE_NAME> \
+                                  ...
+```
+
+{% hint style="info" %}
+You can use `zenml stack register --help` to see a list of all possible 
+arguments to the `zenml stack register` command, including a list of which 
+option to use for which stack component.
+{% endhint %}
+
+#### Activating a Stack
+
+Finally, to start using the stack you just registered, set it as active:
+
+```shell
+zenml stack set <STACK_NAME>
+```
+Now all your code is automatically executed using this stack.
+
+{% hint style="info" %}
+Some advanced stack component flavors might require infrastructure provisioning
```

Comment:
We need to be very careful here. Zenml stack up does not provision infrastructure at all. It just exposes infrastructure which is already deployed locally. Thats its job.

zenml stack recipe deploy might be a different story but we dont have it yet

Reply:
Renamed it to "might require connecting to remote infrastructure components"